lib/timeline/component/ItemSet.js

```diff
@@ -151,9 +151,9 @@ function ItemSet(body, options) {
   this.groupTouchParams = {};
   // create the HTML DOM
 
-  this._create();
-
   this.setOptions(options);
```

Comment:
Are you sure `setOptions` should be called AFTER `_create`? Looks dangerous to me.